### PR TITLE
App or pages

### PR DESCRIPTION
**Pull request by shivamSspirit**
add overall app with basic flow

### CODE REVIEW

File: src/backend/controllers/PlaylistController.js [hunk 1]
```diff
@@ -43,7 +45,12 @@ export const getAllPlaylistsHandler = function (schema, request) {
  * */
 
 export const addNewPlaylistHandler = function (schema, request) {
+  // console.log('req from backend',request)
+
+  // console.log('user',requiresAuth.call(this, request))
+
```
Comment (HeyNitin) [inline src/backend/controllers/PlaylistController.js:51]: remove these

File: src/backend/controllers/PlaylistController.js [hunk 2]
```diff
@@ -107,11 +114,16 @@ export const getVideosFromPlaylistHandler = function (schema, request) {
...
 export const addVideoToPlaylistHandler = function (schema, request) {
   const user = requiresAuth.call(this, request);
+  console.log('user from add vdeos',user)
   if (user) {
+    console.log(request.params)
     const playlistId = request.params.playlistId;
-    const { video } = JSON.parse(request.requestBody);
+    console.log(playlistId)
+    console.log(request.requestBody)
+    const { data } = JSON.parse(request.requestBody);
```
Comment (HeyNitin) [inline src/backend/controllers/PlaylistController.js:123]: remove console logs

**Review by HeyNitin — CHANGES_REQUESTED** [review]
remove console logs, rest is good you can merge

File: src/backend/controllers/WatchLaterController.js [hunk 3]
```diff
@@ -43,10 +43,13 @@ export const getWatchLaterVideosHandler = function (schema, request) {
...
 
 export const addItemToWatchLaterVideos = function (schema, request) {
+  // console.log('req',request)
   const user = requiresAuth.call(this, request);
+  // console.log('iser',user)
   if (user) {
-    const { video } = JSON.parse(request.requestBody);
-    if (user.watchlater.some((item) => item.id === video.id)) {
+    const { data } = JSON.parse(request.requestBody);
+    // console.log(data)
```
Comment (aurobindoGupta) [inline src/backend/controllers/WatchLaterController.js:51]: remove

File: src/backend/db/users.js [hunk 4]
```diff
@@ -9,11 +9,24 @@ import { formatDate } from "../utils/authUtils";
...
+    lastName: "soni",
+    email: "shivam@gmail.com",
+    password: "123456",
     createdAt: formatDate(),
     updatedAt: formatDate(),
   },
 ];
+
+
+// export const users = [
```
Comment (aurobindoGupta) [inline src/backend/db/users.js:22]: remove

**Review by aurobindoGupta — APPROVED** [review]
nice work.

**Pull request merged**